### PR TITLE
Add support for conan advisories

### PR DESCRIPTION
**Pull request by TG1999**

### CODE REVIEW

File: vulnerabilities/package_managers.py [hunk 1]
```diff
@@ -556,6 +556,22 @@ def fetch(self, pkg: str) -> Iterable[PackageVersion]:
             )
 
 
+class ConanVersionAPI(VersionAPI):
+    """
+    Fetch versions of Erlang packages from the hex API
```
Comment (pombredanne) [inline vulnerabilities/package_managers.py:561]: Erlang and hex?

**Review by pombredanne — CHANGES_REQUESTED** [review]
Thanks... See a nit inlined and you need to add a CHANGELOG entry too!

File: vulnerabilities/package_managers.py (hunk 1 above)
Comment (TG1999) [reply vulnerabilities/package_managers.py]: Done!

**Review by pombredanne — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**